MIP0/mip0.md

```diff
@@ -7,287 +11,254 @@ Title: The Maker Improvement Proposal Framework
 
- 1. **Specificity:** A MIP needs to define and address a specific behaviour or single responsibility. MIPs with many different behaviours or responsibilities will not be allowed and must be split up into multiple MIPs.
-	 - This mitigates the risk of having “fine print” or potential attacks hidden in large, complex MIPs.
- 2. **Completeness:** A MIP or MIP Set is complete if it has all the necessary or appropriate parts that cover a whole behaviour and avoids being only a specific part of a greater whole.
-	 - This is important for both understandability, readability and accessibility of MIPs.
-3.  **Avoid overlap:** Multiple MIPs should not implement the same type of behaviour independently. For instance, there should not be two separate but interchangeable ways to do collateral onboarding.
-	- This way the primary and best-understood process for each particular behaviour will be fairly available to everyone, without risking having a knowledge gap that makes it possible for some actors with better access to information to use different and potentially better processes
-4. **Clarity:** A MIP must not have equally valid conflicting interpretations. MIP Authors and MIP Editors must strive to reduce ambiguity. A MIP must be as clear and easy to understand as possible.
-	- Any ambiguous MIPs are likely to cause contention or confusion in the future. Making everything as clear as possible also aids readability and helps to mitigate the risk of hidden attacks.
-5. **Brevity:** A MIP must be as short as possible, including only that which is essential given the other core principles.
-	- The shorter MIPs are the more likely participants in governance are to read them in full. This also serves to reduce the surface area for hidden attacks.
-	
----
+   If the verification is successful, the MIP Editor:
+
+   - Approves the MIP and assigns it a formal MIP number.
+   - Merges in the PR.
 
-### MIP0c3: The MIP Lifecycle
+3. **Request for Comments (RFC)**: A period of reviewing by the community and attendant redrafting begins. The minimum duration of this period is determined by two variables:
 
-**The MIP Lifecycle Flow and MIP Statuses**
+   - `Feedback Period`: 3 months
+   - `Frozen Period`: 1 month
 
-![mip_life_cycle](https://user-images.githubusercontent.com/32653033/79086728-19d93900-7d0b-11ea-8086-c255d919096c.png)
+   These periods can overlap. See `MIP0c1` for their definitions
 
+4. **Fulfilled Feedback Period Requirements:** After the MIP has fulfilled the RFC phase, it is ready for Formal Submission.
 
-**MIP Status Criteria**  
+5. **Formal Submission (FS):** At this point the MIP Author submits their MIP to the Governance Cycle by posting it to the [*Formal Submission* forum category](https://forum.makerdao.com/c/mips/fs/16) within the [formal submission window of a Governance Cycle](https://github.com/makerdao/mips/blob/master/MIP16/mip16.md).
```

Comment:
Moving it to the 'Formal Submission' category?

Reply:
Also, I hate the fact that this is supposed to be done in a 3 day window, but we're not changing actual logic, so lets leave it.